lib/child_process.js

```diff
@@ -195,6 +195,8 @@ exports.execFile = function(file /*, args, options, callback*/) {
 
   var ex = null;
 
+  var cmd = file;
+
```

Comment:
It happens only if `arguments[i]` and `arguments` are mentioned, so it could not be a huge number of cases.